ext/gmp/tests/gmp_cmp.phpt

```diff
@@ -26,7 +26,7 @@ try {
 echo "Done\n";
 ?>
 --EXPECT--
-int(2)
+int(1)
```

Comment:
Maybe instead something like this `$res = gmp_cmp(123123,-123123); <if 64 bits expects 1 or die>...` wdyt ?